tests/src/components/Detail.tsx

```diff
@@ -17,6 +17,15 @@ import Detail, { Props } from "../../../src/components/Detail";
 import { mockStudy } from "../../common/mockdata";
 import userEvent from '@testing-library/user-event';
 
+// Mock react-i18next
+jest.mock('react-i18next', () => ({
```

Comment:
Given that this is a common mock, is there a way to put this in a utility class?

Reply:
Thanks, good idea. I ended up adding a new directory ```tests/mocks/``` for mocks even though it's just ```reacti18nMock.ts``` in it for now.